conans/client/loader.py

```diff
@@ -274,6 +278,8 @@ def _parse_conanfile(conan_file_path):
                     if folder.startswith(current_dir):
                         module = sys.modules.pop(added)
                         sys.modules["%s.%s" % (module_id, added)] = module
+    except ConanException:
```

Comment:
There is nothing that can launch this, is there?

Reply:
This is catching any `ConanException` during the loading of the `ConanFile`, for example, a _not found_ python requires, like [here](https://github.com/conan-io/conan/pull/4894/files#diff-ca33fe4baee1eb978f1054011f842db3R74) (there I'm reraising adding context to the `NotFoundException`).